app/views/admin/feedback_messages/_feedback_message.html.erb

```diff
@@ -210,7 +210,7 @@
    formData.append('id', id);
    formData.append('status', statusSelectTag.options[statusSelectTag.selectedIndex].value);
 
-   fetch('/admin/reports/save_status', {
+   fetch(statusBtn.dataset.path, {
```

Comment:
I think this is where we could inline the path helpers. Non blocking if I'm missing something or mistaken (could totally be the case 😄)

Reply:
Ah @fdoxyz you mean replacing `statusBtn.dataset.path` with `save_status_admin_reports_path` and removing the data attribute. I tried this and the js doesnt seem to know what  `save_status_admin_reports_path` is. I even tried it by directly accessing `Rails.application.routes.url_helpers.send_email_admin_reports_path` with no luck. I also thought the js might have access to the path helpers as the js is scripted in the same file. Curious to know why it doesnt work.

Reply:
@Ridhwana being JS that's inline a ERB template, I think it would have worked with `fetch("<%= save_status_admin_reports_path")`

Reply:
Ah its the erb tags that I may have been missing. I'll try it out in a bit.